neurom/core/dataformat.py

```diff
@@ -29,13 +29,13 @@
 """Data format definitions."""
 
 
-_COL_COUNT = 7
+_COL_COUNT = 4
 
 
 class COLS:
     """Column labels for internal data representation."""
     COL_COUNT = _COL_COUNT
-    (X, Y, Z, R, TYPE, ID, P) = range(_COL_COUNT)
+    (X, Y, Z, R) = range(_COL_COUNT)
```

Comment:
For an enum like object like this one I would prefer to see : 
```
X = 0
Y = 1
...
```
even if more verbose

Reply:
Lets stick here to v2 migration only, and postpone general code cleaning for the next PR.